### ISSUE TITLE
host crash with LX branded zone running

### ISSUE THREAD

**Issue by nirv199** [joyent/smartos-live#574]
[root@headnode (BASEMENT) /var/crash/volatile]# mdb -e '::status; $C' 0
debugging crash dump vmcore.0 (64-bit) from headnode
operating system: 5.11 joyent_20151223T094457Z (i86pc)
image uuid: (not set)
panic message: 
BAD TRAP: type=e (#pf Page fault) rp=ffffff002f332940 addr=0 occurred in module 
"FSS" due to a NULL pointer dereference
dump content: kernel pages only
ffffff002f332a70 fss_update_list+0x7a(3)
ffffff002f332ab0 fss_update+0x52(0)
ffffff002f332b00 callout_list_expire+0x98(ffffff06f521d9c0, ffffff0819be3700)
ffffff002f332b30 callout_expire+0x3b(ffffff06f521d9c0)
ffffff002f332b60 callout_execute+0x20(ffffff06f521d9c0)
ffffff002f332c20 taskq_thread+0x2d0(ffffff06f6dc5b18)
ffffff002f332c30 thread_start+8()
[root@headnode (BASEMENT) /var/crash/volatile]#

**Comment by bcantrill** [comment]
Apologies for the delay in looking at this; can you make available the crash dump here?

**Comment by nirv199** [comment]
Hey Bryan! I am no longer seeing the issue since the 20160303T211841Z build (got the SDC image from https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/usb-latest.tgz ).

I remember I saw the same issue till the very previous build to the 20160303T211841Z one (I am sorry I didn't keep track of the predecessor version/build). This was a brand new server and I was doing full SDC re-installs on each attempt.

I was running the exact same commands to get that LX zone up and running with the software packages I needed; and the crash would always happen during packages installation inside that LX zone (which led me to believe this could be either a file system or network-interface related syscall thing (it was downloading required packages from the internet) as CPU was barely doing anything - no other user VM apart from this one and the system required ones).

So far so good, I think this issue was addressed by one of the fixes/changes between 20160303T211841Z and its predecessor build. System has been stable ever since: 13:44:00    up 12 day(s), 23:38,  2 users,  load average: 3.56, 3.60, 3.61

I need to apologize I had to re-image the removable media I had the dump on (losing the data) a week after the issue was gone. Yeah, I need to do a Costco run to get those big packages of USB drives, sorry :)
